src/current/_includes/releases/v24.1/v24.1.2.md

```diff
@@ -0,0 +1,71 @@
+- Fixed an issue where [adding a column](https://www.cockroachlabs.com/docs/v24.1/alter-table#add-column) with a default value of an empty [array](https://www.cockroachlabs.com/docs/v24.1/array) would not succeed. [#125325][#125325]
+- [`ALTER TABLE ... ADD CONSTRAINT UNIQUE`](https://www.cockroachlabs.com/docs/v24.1/alter-table) will now fail with a well-formed error message and code `42601` if a statement tries to add a [unique constraint](https://www.cockroachlabs.com/docs/v24.1/unique) on an expression. [#125417][#125417]
+- Fixed a bug where the [`public` schema](https://www.cockroachlabs.com/docs/v24.1/schema-design-overview#schemas) would be created with the wrong owner. Previously the [`admin` role](https://www.cockroachlabs.com/docs/v24.1/security-reference/authorization#roles) would own the `public` schema. Now, the database owner is also the owner of the `public` schema. The ownership can be altered after the schema is created. [#125533][#125533]
+- Fixed a bug in v24.1, v23.2, and v23.1 where using the `changefeed.aggregator.flush_jitter` [cluster setting](https://www.cockroachlabs.com/docs/v24.1/cluster-settings#setting-changefeed-aggregator-flush-jitter) with the [`min_checkpoint_frequency`](https://www.cockroachlabs.com/docs/v24.1/create-changefeed#min-checkpoint-frequency) option set to `0` could cause panics. [#125459][#125459]
+- Resolved a concerning log message that said `"expiration of liveness record ... is not greater than expiration of the previous lease ... after liveness heartbeat"`. This message is no longer possible. [#125449][#125449]
+- Previously, CockroachDB would hit an internal error when evaluating [`INSERT`s](https://www.cockroachlabs.com/docs/v24.1/insert) into [`REGIONAL BY ROW`](https://www.cockroachlabs.com/docs/v24.1/set-locality#set-the-table-locality-to-regional-by-row) tables where the source is a `VALUES` clause with a single row and at least one boolean expression; this is now fixed. The bug was introduced in the v23.2.0 release. [#126209][#126209]
+
+<h3 id="v24-1-2-performance-improvements">Performance improvements</h3>
+
+- Make error handling in the [vectorized execution](https://www.cockroachlabs.com/docs/v24.1/vectorized-execution) engine much cheaper. This should help avoid bad metastable regimes perpetuated by statement timeout handling consuming all CPU time, leading to more statement timeouts. [#124996][#124996]
+- Some [privilege](https://www.cockroachlabs.com/docs/v24.1/security-reference/authorization#managing-privileges) checks when scanning the `crdb_internal.system_jobs` internal table now happen once before the scan, instead of once for each row. This will improve performance for queries that read from `crdb_internal.system_jobs`. [#125250][#125250]
```

Comment:
```suggestion
- Some [privilege](https://www.cockroachlabs.com/docs/v24.1/security-reference/authorization#managing-privileges) checks when scanning the `crdb_internal.system_jobs` internal table now happen once before the scan, instead of once for each row. This improves performance for queries that read from `crdb_internal.system_jobs`. [#125250][#125250]
```

Reply:
✅